src/bin/travis/testCentaurCwlConformancePAPI.sh

```diff
@@ -85,6 +85,7 @@ java \
   -Xmx2g \
   -Dconfig.file="$JES_CONF" \
   -Dsystem.new-workflow-poll-rate=1 \
+  -Dsystem.job-shell=/bin/sh \
```

Comment:
As there is already a `-Dconfig.file` used on this command line, this configuration should be kept together with the rest inside the file `$JES_CONF` aka `src/bin/travis/resources/jes_centaur.conf.ctmpl`.